r/run_model.R

```diff
@@ -13,12 +29,12 @@ opt <- parse_args(opt_parser)
 
 
 # Read in input file ------------------------------
-data <- read_csv(file.path(opt[["input-dir"]], "names.csv"), col_types = "c")
+data <- read_csv(file.path(opt[["input-dir"]], "data.csv"), col_types = "c")
```

Comment:
```suggestion
data <- read_csv(file.path(opt[["input-dir"]], "data.csv"), show_col_types = FALSE)
```
Using `col_types = "c"` converts all column data types to character - which might not be what we want.

Reply:
Ah good point! Thanks, @rrchai